src/main/java/mate/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final String SESSION_ATTRIBUTE_ID = "id";
```

Comment:
```suggestion
    private static final String SESSION_ATTRIBUTE_ID = "driver_id";
```

just `id` it's not very informative (fix in all places)

Reply:
fixed